bot/exts/moderation/infraction/management.py

```diff
@@ -262,6 +262,44 @@ async def search_reason(self, ctx: Context, reason: str) -> None:
         )
         await self.send_infraction_list(ctx, embed, infraction_list)
 
+    # endregion
+    # region: Search infractions by given user
+    @infraction_group.command(name="by", aliases=("b",))
+    async def search_by_user(
+        self,
+        ctx: Context,
+        user: t.Union[discord.Member, t.Literal["m", "me"]],
+        oldest_first: bool = False
+    ) -> None:
+        """
+        Search for infractions made by `user`.
+
+        Use "m" or "me" as the `user` to get infractions by author.
+
+        Use "1" for `oldest_first` to send oldest infractions first.
+        """
+        if isinstance(user, discord.Member):
+            moderator_id = user.id
+            moderator_name_discrim = str(user)
+        else:
+            moderator_id = ctx.author.id
+            moderator_name_discrim = str(ctx.author)
+
+        infraction_list = await self.bot.api_client.get(
+            'bot/infractions/expanded',
+            params={
+                'actor__id': str(moderator_id),
+                'ordering': f'{["-", ""][oldest_first]}inserted_at'  # `'inserted_at'` makes api return oldest first
+            }
+        )
+
+        embed = discord.Embed(
+            title=f"Infractions by `{moderator_name_discrim}` (`{moderator_id}`)",
+            colour=discord.Colour.orange()
+        )
```

Comment:
This might seem a bit of a nitpick, but I think it's not necessary to create variables for the ID and the name+discrim.

You can have this condition at the beginning:
```py
if actor in ("m", "me"):
    actor = ctx.author
```
And then have `'actor__id': str(actor.id),` instead of  `'actor__id': str(actor_id),`, as it doesn't seem like you save anything with that variable.

Then below you can have
```py
title=f"Infractions by `{actor}` (`{actor.id}`)",
```
Since as you've shown at the beginning of the function, `str(user)` gives the name and discrim, and you only use it in this one place anyway.